core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/RegistryResources.java

```diff
@@ -160,6 +160,23 @@ public static final class SecurityManagement {
         public static final String SERVER_INTERNAL_PRIVATE_KEY_PASSWORD = "Security.InternalKeyStore.KeyPassword";
         public static final String SERVER_INTERNAL_KEYSTORE_TYPE = "Security.InternalKeyStore.Type";
 
+        // Custom KeyStore related constants.
+        public static final class CustomKeyStore {
+            public static final String KEYSTORE_PREFIX = "CUSTOM/";
```

Comment:
missing new line

Reply:
```suggestion
            public static final String CUSTOM_KEYSTORE_PREFIX = "CUSTOM/";
```

Reply:
FIxed in https://github.com/wso2/carbon-kernel/pull/4025/commits/3aa02c8fd1024563e7aefe9329049af4802dfac3